### PR TITLE
Recipe changes for (dis)enchanter

### PR DESCRIPTION
**Pull request by ghost**
## Short Description
Changed crafting recipe of advanced disenchanter from auto disenchanter 1 to auto disenchanter 2
Changed crafting recipe of advanced enchanter from auto enchanter 1 to auto enchanter 2
Mostly auto enchanter 2 and auto disenchanter 2 is not used much because the crafting recipe of advanced enchanter and advanced disenchanter is much cheaper [Direct from 1 -> Advanced and no progressions]
This changes crafting recipe to follow a progression auto enchanter 1 -> auto enchanter 2 -> advanced enchanter
This changes crafting recipe to follow a progression auto disenchanter 1 -> auto disenchanter 2 -> advanced disenchanter
Also would help servers who want a progression system 

## Related Issues
N/A

## Video Proof
I can not record but will post a few pictures to show that it works
First proof that there are no startup errors -> https://paste.walshy.dev/iDlCDF0lf9e0
image
image
image
image
image
image
image
image
image
image
image
image
image

## Checklist
- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with base Slimefun and made sure nothing breaks/unexpected happens.

### CODE REVIEW

**Review by FN-FAL113 — APPROVED** [review]
LGTM

**Review by Sefiraat — CHANGES_REQUESTED** [review]
Looking good to me, could you please update the changelog with this PR however, just as it's something Riley kept up and it'd be a good idea to keep it running :)

**Comment by FN-FAL113** [comment]
> Do I need to do something else or wut?

Wait for sefi's approval

**Pull request closed without merging**
